CONTRIBUTING.md

```diff
@@ -1,13 +1,25 @@
 # Contribution
 
 ## Table of Contents
+* [Permission levels](#permissions)
 * [Labels](#labels)
 * [Issues & Labels](#issues)
 * [Pull Requests](#prs)
 * [Pull Requests Approval Process](#approvals)
 * [Idle Pull Requests Policy](#idle)
 * [Unresolved Pull Requests Policy](#unresolved)
 
+<a name="permissions"></a>
+### Permission levels
+There are different permissions (levels) available for contributing into CNTT CNTT:
```

Comment:
Typo: Two occurrences of CNTT.

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/1401/commits/55b7a67207923f077dfd75d040ddeca6a7127a32